### PR TITLE
Workflows documentation update for Storybook start/build with --docs flag

### PR DESCRIPTION
**Pull request by jonniebigodes**
This pull request adds a bit more context to the workflows section, more specifically to the "Publish Storybook" page. 

The appropriate flag is already added in the pull request that covering the updates for that section, in here #11751

Feel free to provide feedback

### CODE REVIEW

File: docs/workflows/publish-storybook.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ Storybook will create a static web application at the path you specify. This can
...
+
+Asides from the `-o` flag, you can also include other flags to build Storybook, for instance if you're using [Docs](../writing-docs/introduction.md), you can append the `--docs` flag and Storybook will build your [MDX](../writing-docs/mdx.md) and [CSF](../writing-stories/introduction.md#component-story-format) stories into a rich and interactive documentation.
+
+You can learn more about these flag options [here](../api/cli-options.md).
+
+
+</div>
+
+
+If you need to build
```
Comment (tmeasday) [inline docs/workflows/publish-storybook.md:31]: Trailing sentence?

**Review by tmeasday — APPROVED** [review]
Looks good apart from leftover sentence?

File: docs/workflows/publish-storybook.md (hunk 1 above)
Comment (jonniebigodes) [reply docs/workflows/publish-storybook.md]: That sentence wasn't meant to be there. I've removed it with the latest commit.

**Comment by jonniebigodes** [comment]
merging as is, updates will be made with follow up pull requests.

**Pull request merged**